From 1ed10e612ae2203acbec4392505d703eba67ddf5 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Thu, 18 Apr 2024 10:57:38 +0100
Subject: [PATCH] Disable contact maps and remove libchamplain build-dependency

clutter-1.0 is unmaintained upstream (#996690) and should ideally not
be included in trixie.

Closes: #659512, #1018120
---
 debian/control | 2 --
 debian/rules   | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/debian/control b/debian/control
index 8c9f3a3eaa..b80153c0b6 100644
--- a/debian/control
+++ b/debian/control
@@ -7,7 +7,6 @@ Build-Depends: cmake,
                debhelper-compat (= 13),
                dh-sequence-gnome,
                dpkg-dev (>= 1.16.1),
-               libchamplain-gtk-0.12-dev (>= 0.12.21),
                libglib2.0-dev (>= 2.66),
                libgtk-3-dev (>= 3.10.0),
                libgail-3-dev (>= 3.0.2),
@@ -51,7 +50,6 @@ Build-Depends: cmake,
                libsm-dev,
                libice-dev,
                gsettings-desktop-schemas-dev (>= 2.91.92),
-               libclutter-gtk-1.0-dev (>= 0.90),
                highlight,
                libcryptui-dev,
                libgnome-autoar-0-dev (>= 0.1.1),
diff --git a/debian/rules b/debian/rules
index 324a55f4cb..ce64fb91b8 100755
--- a/debian/rules
+++ b/debian/rules
@@ -20,7 +20,7 @@ override_dh_auto_configure:
 		-DWITH_OPENLDAP=ON \
 		-DENABLE_PLUGINS=all \
 		-DENABLE_PST_IMPORT=ON \
-		-DENABLE_CONTACT_MAPS=ON \
+		-DENABLE_CONTACT_MAPS=OFF \
 		-DENABLE_AUTOAR=ON \
 		-DENABLE_INSTALLED_TESTS=OFF \
 		-DWITH_HELP=ON \
-- 
2.39.2

